daemon/src/engine/qcommon/files.cpp

```diff
@@ -307,12 +307,13 @@ int FS_Seek(fileHandle_t handle, long offset, int origin)
 	}
 }
 
+// Unused
 void FS_ForceFlush(fileHandle_t handle)
 {
 	FS_CheckHandle(handle, true);
 	handleTable[handle].forceFlush = true;
 }
-
+// Unused
```

Comment:
I'd say these can be removed, as well as the forceFlush member and related code. Let's ask @Amanieu for confirmation.